wordpress-mobile/WordPress-Android
# Jetpack: Update network and db calls to adding various missing site settings

This PR is #6741's sibling. It updates network serialization methods for fetching and updating the new options. It also adds columns to the database and handles migration.

To test:
* Since there's no way to see these changes in the app I'd just set a breakpoint [here](https://github.com/wordpress-mobile/WordPress-Android/compare/issue/6652-missing-settings-master...issue/6652-missing-settings-network#diff-c7e7383fc8aa13adeb5e7f0a75d559b2R208) and make sure all the values are set correctly.
